src/main/java/Duke.java

```diff
@@ -6,70 +6,81 @@ public class Duke {
 
 
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo + "\n" + "How may i help you?\n");
+        System.out.println("Hello from mars.\n" + "How may i help you?\n");
 
         Scanner sc = new Scanner(System.in);
 
         String echo = sc.nextLine();
 
         List<Task> store = new ArrayList<>();
 
-        while (!echo.equals("bye")) {
+        while (true) {
             if (echo.equals("list")) {
                 for (int i = 0; i < store.size(); i++) {
                     System.out.print(i + 1 + ":" + store.get(i) + "\n");
                 }
                 echo = sc.nextLine();
-            } else if ((echo.length() > 4) && (echo.substring(0,4).equals("mark"))) {
+            } else if (echo.equals("bye")) {
```

Comment:
I like the change of code in identifying the type of command, makes the entire code clean and neat.